### PR TITLE
Test on Mono

### PR DESCRIPTION
**Pull request by sideeffffect**
add a testing scheme in Travis which uses Mono
(still has to be enabled in Travis)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # F# |> BABEL
 
 ### The compiler that emits JavaScript you can be proud of. [Try it out!](http://fsprojects.github.io/Fable/repl.html)
-[![Build status](https://ci.appveyor.com/api/projects/status/vlmyxg64my74sik5?svg=true)](https://ci.appveyor.com/project/alfonsogarciacaro/fable)
+[![Mono Build Status](https://travis-ci.org/fsprojects/Fable.svg "Mono Build Status")](https://travis-ci.org/fsprojects/Fable) [![.NET Build Status](https://ci.appveyor.com/api/projects/status/vlmyxg64my74sik5?svg=true ".NET Build Status")](https://ci.appveyor.com/project/alfonsogarciacaro/fable)
```
Comment (alfonsogarciacaro) [inline README.md]: It would be better to keep both badges: AppVeyor and Travis. Can you please put the Appveyor's one back?

**Comment by alfonsogarciacaro** [comment]
That would be a very nice addition, thanks! Can you please check my comment above and fix the PR before merging it? Cheers!

**Comment by davidpodhola** [comment]
Ondra, this is great, thanks!

File: README.md (hunk 1 above)
Comment (davidpodhola) [inline README.md]: @alfonsogarciacaro I think the diff just visualize it wrong. I tried to merge manually and I still have both badges.

capture

Comment (alfonsogarciacaro) [inline README.md]: You're right! Sorry, my bad :)

**Pull request merged**